cli/asc.js

```diff
@@ -655,6 +655,7 @@ exports.main = function main(argv, options, callback) {
     stats.validateCount++;
     stats.validateTime += measure(() => {
       if (!module.validate()) {
+        console.log(module.toText())
```

Comment:
this makes my life 10000x easier by printing out the entire module when validation fails. Might remove it before we merge back into upstream, but I think I'm also going to suggest it as a change to the main repo